### PR TITLE
Fix CAPI config folder location

### PR DESCRIPTION
**Pull request by Sunnatillo**
This PR fixes CAPI config folder location for tilt env and host_cleanup.sh file

### CODE REVIEW

**Comment by Sunnatillo** [comment]
/test-centos-integration-release-1-5
db locked flake

**Review by lentzi90 — COMMENTED** [review]
/approve

File: host_cleanup.sh [hunk 1]
```diff
@@ -63,4 +63,6 @@ fi
 # Clean up any serial logs
 sudo rm -rf /var/log/libvirt/qemu/*serial0.log*
 
+# TODO(Sunnatillo): Remove first line after deprication of camp3 v1.4
 rm -rf "${HOME}"/.cluster-api
+rm -rf "${HOME}"/.config/cluster-api
```
Comment (tuminoid) [inline host_cleanup.sh:68]: Should this not have XDG_CONFIG_FOLDER variable instead of hardcoding it?

**Review by tuminoid — COMMENTED** [review]
/lgtm

**Pull request merged**